projects/common/src/utilities/validators/form-validators.ts

```diff
@@ -0,0 +1,13 @@
+import { AbstractControl } from '@angular/forms';
+import { getStringsFromCommaSeparatedList } from '../formatters/string/string-formatter';
+import { areEmailAddressesValid } from './email-validator';
+
+// tslint:disable-next-line: no-namespace
+export namespace CustomFormValidators {
```

Comment:
In the ideal scenario, we can have something like @hypertrace/forms and use `HtValidators ` or `CustomValidators` to not use "Form" in between

Reply:
It is meant for validators in forms. So that's why I think `CustomFormValidators` is the right name

Reply:
That depends, on how you look at it, validators in angular forms are not named `FormValidators`, they are just `Validators`. That's understandable because we're importing from `@angular/forms`. SO Currently it is fine, because we're importing from `@hypertrace/common`

Reply:
Agreed, if we are able to package it in `@hypertrace/forms`, we can export it as `HtValidators`.

Speaking of which, is it a lot of work to setup a project like `@hypertrace/forms`, or something preventing us from doing it now (complexity setting it up?). Then we don't have to make breaking changes (moving from `@hypertrace/common` to `@hypertrace/forms` later on).